src/routes/reports.rs

```diff
@@ -64,32 +64,71 @@ pub async fn report_create(
     };
 
     match new_report.item_type {
-        ItemType::Project => {
-            report.project_id = Some(
-                serde_json::from_str::<ProjectId>(&format!(
-                    "\"{}\"",
-                    new_report.item_id
-                ))?
-                .into(),
+        ItemType::Project | ItemType::Mod => {
+            let project_id = serde_json::from_str::<ProjectId>(&format!(
+                "\"{}\"",
+                new_report.item_id
+            ))?;
```

Comment:
I know this was in the code before, but we should just use `parse_base62` it's a lot better for all of these